pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -61,12 +61,15 @@ const (
 )
 
 var (
-	// The container that we use to run in the EventListener Pods
-	elImage = flag.String("el-image", "override-with-el:latest",
+	// ELImage defines the container that we use to run in the EventListener Pods
```

Comment:
I renamed this flag to capitalize it like the other flags. Mainly for consistency.